index.bs

```diff
@@ -39,6 +38,7 @@ Metadata Order: !*, *, This version
 <pre class="anchors">
 spec: HTTP; urlPrefix: https://httpwg.org/specs/rfc7231.html#
     type: dfn; text: HTTP 3xx statuses; url: status.3xx
+    type: dfn; text: bc-traversable; url: bc-traversable
```

Comment:
I think this was from my original PR.  I guess I put it under http spec instead of html.  I was trying to link to:

https://html.spec.whatwg.org/#top-level-traversables

Reply:
https://html.spec.whatwg.org/multipage/document-sequences.html#bc-traversable will be [=browsing context/top-level traversable=].